include/shared/rpc/RPCRequests.h

```diff
@@ -203,23 +203,15 @@ operator<<(std::ostream &os, const RecordLookUpResponse::RecordError &re)
 inline std::ostream &
 operator<<(std::ostream &os, const JSONRPCError &err)
 {
-  os << "Error found.\n";
-  os << "code: " << err.code << '\n';
-  os << "message: " << err.message << '\n';
+  os << "Server Error found:\n";
+  os << "[" << err.code << "] " << err.message << '\n';
   if (err.data.size() > 0) {
-    os << "---\nAdditional error information found:\n";
-    auto my_print = [&](auto const &e) {
-      os << "+ code: " << e.first << '\n';
-      os << "+ message: " << e.second << '\n';
-    };
-
     auto iter = std::begin(err.data);
 
-    my_print(*iter);
+    os << "- [" << iter->first << "] " << iter->second << '\n';
     ++iter;
     for (; iter != std::end(err.data); ++iter) {
-      os << "---\n";
-      my_print(*iter);
+      os << "- [" << iter->first << "] " << iter->second << '\n';
```

Comment:
Since the interleaved `---` is no longer being printed, this could just be a simple `for (auto it: err.data) ...` loop.

Reply:
True.  Missed that, thanks a lot, fixed now.